ergoCubSN000/hardware/motorControl/head-eb20-j0_1-mc.xml

```diff
@@ -47,9 +47,9 @@
         <param name="outputType">               pwm             </param>
         <param name="fbkControlUnits">          metric_units    </param>
         <param name="outputControlUnits">       machine_units   </param>
-        <param name="kp">                       -512        +535        </param>
-        <param name="kd">                       -17         +14         </param>
-        <param name="ki">                       -1621       +1670       </param>
+        <param name="kp">                       -150        +150        </param>
+        <param name="kd">                       0           0         </param>
+        <param name="ki">                       -15         +15       </param>
```

Comment:
We're retuning the PID using the digital twin. To this end, we still need references to our old gains. Please, remove these changes.

Reply:
On a second thought, we could accept these very detuned gains to then modify them later.